packages/eslint-plugin/src/rules/consistent-type-imports.ts

```diff
@@ -603,15 +693,29 @@ export default util.createRule<Options, MessageIds>({
           if (sourceImports.typeOnlyNamedImport.range[1] <= node.range[0]) {
             yield insertTypeNamedSpecifiers;
           } else {
+            // TODO: delay inserting b/c....
```

Comment:
b/c...?

Reply:
ack that was me just learning the rules.  Was hoping to ask a question about the patterns.  But we simplified this PR greatly once we moved a ton of logic to eslint-plugin-import.  Been so long, I forgot as well.